design-proposals/emts-scale-provisioning.md

```diff
@@ -192,20 +182,28 @@ that is almost equal to Tinker action doing the same.
 Ideally, EMT-S OS installation scripts should act as a "standalone Tink worker" that reads Tinkerbell YAML file and executes all steps as defined there.
 We won't be there yet for 3.1, but this should be a direction for us, so that we don't have diverged codebases.
 
-### EIM-only profile of EMF deployment (EIM-S)
+### Detach of Edge Nodes
 
-The provisioning of EMT-S at scale will be driven by a local orchestrator instance that only includes EIM components. 
-The main requirement of EIM standalone is to make it easy and resource-efficient for customers to deploy.
-
-For MVP, we will leverage the EIM Standalone profile. This is an EMF deployment profile that only deploys EIM with Observability and FPS components.
+Once ENs are provisioned and ready to be shipped to the field to operate as standalone ENs, they should be detached from the local EMF orchestrator.
+An explicit detach would be needed if there are agents' configurations, certificates, credentials, etc. installed on ENs.
+However, **BM agents won't be configured and activated for EMT-S OS profiles**. Therefore, the only operation to detach ENs from the orchestrator is to delete them via northbound API.
```

Comment:
On the orchestrator side, we should mention how this interacts with revocation of the auth credentials in Keycloak when an EN is removed, or point to docs that talk about this.